### PR TITLE
Add moment page 11 for VPN [fix #10737]

### LINKED ISSUE #10737: 11/29 VPN Moments page
## Description

Part of Epic https://github.com/mozilla/bedrock/issues/10702

[Copy Document](https://docs.google.com/document/d/1_TNRdo4t0yrNO5RBfSPTRrCkxEJ94rGHAKdEgS31Fp0/edit?usp=sharing)

Designs: https://www.figma.com/file/UFZ8R9V3lVxpZFQ34g0fN9/VPN-Moments

Slack Channel: `#www-moments`

Go Live Date: 11/29
---

#### Success Criteria
- [ ] Localize Page
- [ ] Develop Page
- [ ] Go live 11/29

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds a new welcome/moment page promoting Mozilla VPN.

Target locales are en-US, de, fr, it, nl, es-ES, and en-GB, some of which are handled through Smartling and the rest through Pontoon.

## Issue / Bugzilla link
#10737 

## Testing
http://localhost:8000/firefox/welcome/11/
Test in both light mode and dark mode and make sure it looks right in both, including some changing colors in the SVGs.

### CODE REVIEW

**Comment by craigcook** [comment]
Do Not Merge for now because copy is still pending final signoff by legal.

File: l10n/configs/pontoon.toml
Comment (craigcook) [inline l10n/configs/pontoon.toml:198]: @peiying2 The full list of target locales is en-US, de, fr, it, nl, es-ES, and en-GB. Of these, I believe de and fr are automatically sent to Smartling so I've only listed it, nl, and es-ES here (same as page10 above). I don't think we need to list en-GB if it can just display the same en file as en-US.

So does this configuration look correct? Do I need to add anything to `vendor.toml` as well, or does it just get ALL Fluent files by default?

Comment (peiying2) [reply l10n/configs/pontoon.toml:198]: This looks good!

**Review by alexgibson — APPROVED** [review]
r+ to merge when ready 🌵

**Comment by craigcook** [comment]
Waiting for a new third blurb, or else a decision to go with only two if we can't come up with a third. So this is blocked pending that decision.

**Comment by craigcook** [comment]
Ended up with a different first blurb, and swapping images between the first and second. We have signoff on the new copy so this is good to merge.

**Review by nathan-barrett — APPROVED** [review]
r+ 🧄

**Pull request merged**
